app/config/parameters.yml.dist

```diff
@@ -8,12 +8,19 @@ parameters:
     database_path:     "%kernel.root_dir%/app.sqlite"
 
     phpcr_backend:
-        # use Jackalope Doctrine DBAL
+        # if you're using Jackalope Doctrine DBAL
         type: doctrinedbal
         connection: default
         caches:
             meta: doctrine_cache.providers.phpcr_meta
             nodes: doctrine_cache.providers.phpcr_nodes
+        
+        # if you're using Jackalope Jackrabbit, comment the lines above
+        # and uncomment the lines after this comment
+        # logging: true
+        # profiling: true
```

Comment:
this is not specific to jackalope-jackrabbit.

we should add logging and profiling after type and url, with its own comment: "You can enable logging and profiling with the symfony toolbar by uncommenting the respective lines below".